vmdb/app/models/host.rb

```diff
@@ -1400,6 +1399,14 @@ def connect_ssh(options={})
       rl_user, rl_password = self.auth_user_pwd(:root)
       su_user, su_password = nil, nil
     end
+    return rl_user, rl_password, su_user, su_password, {}
+  end
+
+  def connect_ssh(options={})
+    require 'MiqSshUtil'
+
+    rl_user, rl_password, su_user, su_password, additional_options = ssh_users_and_passwords
```

Comment:
Nice @Ladas... hopefully this refactoring will allow others as it seems like that method is still doing too much but at least it's a single method.  But we do need tests for this `ssh_users_and_passwords` on `Host` vs. `HostOpenstackInfra`